### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by VikramTiwari**
for https://github.com/linnovate/meanio-system/issues/7

### CODE REVIEW

**Comment by VikramTiwari** [comment]
This is also causing new MEAN builds to fail. Labeling Bug.

**Comment by timelf123** [comment]
LGTM, though maybe set to node 6? I know some people haven't had the chance to upgrade

**Comment by adskjohn** [comment]
node 4.x is still under LTS so it's not unreasonable that people may be using it.  Would >=4.0.0 fix the issue here?

File: package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
   },  
   "mean": "0.6.x",
   "engines": {
-    "node": "4.x",
-    "npm": "3.x"
+    "node": "7.x",
+    "npm": "4.x"
```
Comment (timelf123) [inline package.json]: `">=3.x"`

File: package.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
   },  
   "mean": "0.6.x",
   "engines": {
-    "node": "4.x",
-    "npm": "3.x"
+    "node": "7.x",
```
Comment (timelf123) [inline package.json]: `">=4.0.0"`

**Review by timelf123 — CHANGES_REQUESTED** [review]

**Comment by VikramTiwari** [comment]
Sorry guys, it's been busy few weeks. Fixed.

**Review by timelf123 — APPROVED** [review]

**Pull request merged**